### ISSUE TITLE
POB Crashes when loading Scion build

### ISSUE THREAD

**Issue by Arychamel** [PathOfBuildingCommunity/PathOfBuilding#6455]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### Screenshots of crash/graphical issue, version and options menu

![Crash](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/28871596/4e8de822-d4d1-4c21-bfb5-176d74b24614)

### What are your system specifications and configuration?

Windows 10
AMD Radeon RX 5700XT Red Devil
Driver 31.0.12027.9001
"C:\Users\tiama\AppData\Roaming\Path of Building Community\Path of Building.exe" "C:\Users\tiama\AppData\Roaming\Path of Building Community\Launch.lua"

### How to reproduce the issue

Load up a Scion build I have saved in POB.
This appears to be the only one that triggers the crash.

### Character build code

```shell
https://gist.github.com/Arychamel/077625673eca7668dbb00ab6cda0b0b5

Pastebin wouldn't work, but here is the XML code for the build that crashes POB.
```

**Comment by Paliak** [comment]
Fixed by https://github.com/PathOfBuildingCommunity/PathOfBuilding/pull/6452 will be solved by next hotfix.

**Comment by LocalIdentity** [comment]
Fixed in 2.33.0
